app/assets/stylesheets/images.scss

```diff
@@ -9,7 +16,7 @@
 .image-topbar-mobile {
   margin-right: 0.2rem;
   border-radius: 2rem;
-  border: 0.05rem solid white;
+  border: 0.1rem solid white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.